packages/webapp/pages/_app.tsx

```diff
@@ -81,6 +82,9 @@ function InternalApp({ Component, pageProps, router }: AppProps): ReactElement {
   useInAppNotification();
   const { modal, closeModal } = useLazyModal();
   usePrompt();
+
+  useQuery(EXTENSION_PERMISSION_KEY, () => ({}));
```

Comment:
Wait we reset it to empty?
Can't we make it cleaner, this seems very viable to someone removing it.

Reply:
I will add a note here explaining what this does, however, keep in mind that this is not the long term solution, I am completely open to working on a cleaner way to manage these things in a separate PR if you want.